samples/wifi/thread_coex/README.rst

```diff
@@ -186,11 +186,11 @@ Testing
       1050759502         /dev/ttyACM3    VCOM1
       $
 
-	  In this example, ``1050779496`` is the device ID of the first nRF7002 DK and ``1050759502`` is the device ID of the other one.
```

Comment:
The numbering breaks when there is no indent:
![image](https://github.com/user-attachments/assets/38c86b99-d990-474b-880d-4c2fcc16bcc9)
I believe these parts should be indented by three spaces so they are inside the numbered step, but not inside the code block.